### PR TITLE
Clean tests

### PR DESCRIPTION
**Pull request by rluvaton**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

----

I'll clean the rest of the tests, just wanted to know if it's a desirable change, no logic has been changed

For easier review, I advise you to check the _Hide whitespace_ checkbox

image

### CODE REVIEW

File: test/global-rate-limit.test.js [hunk 1]
```diff
@@ -6,56 +6,52 @@ const Redis = require('ioredis')
...
+  t.equal(res.headers['x-ratelimit-remaining'], 0)
+  t.equal(res.headers['retry-after'], 1000)
+  t.same({
+    statusCode: 429,
+    error: 'Too Many Requests',
+    message: 'Rate limit exceeded, retry in 1 second'
+  }, JSON.parse(res.payload))
+
+  // TODO - use sinom timers
+  await sleep(1100)
```
Comment (rluvaton) [inline test/global-rate-limit.test.js:48]: Used this and not sinon timers to keep the current behavior of the tests

**Comment by rluvaton** [comment]
The failed test is not even the file I'm editing 🤷🏻‍♂️

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by mcollina** [comment]
Thank you very much for the help!!!

**Comment by rluvaton** [comment]
> Thank you very much for the help!!!


You're welcome!, always love to help 😄

**Pull request merged**